includes/class-main.php

```
@@ -143,6 +143,10 @@ private function includes(): void {
 		require_once WPPO_PLUGIN_PATH . 'includes/class-core-tweaks.php';
 		require_once WPPO_PLUGIN_PATH . 'includes/class-object-cache.php';
 
+		// Phase 1 — Local Diagnostics (v1.5.0).
+		require_once WPPO_PLUGIN_PATH . 'includes/class-telemetry.php';
```

PHPStan (2.1.46)

[warning] Constant WPPO_PLUGIN_PATH not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)